src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -131,6 +131,16 @@ enum xgq_cmd_log_page_type {
 	XGQ_CMD_LOG_BACKUP_FPT		 = 0xc,
 };
 
+/*
+ * VMR SC status defines
+ */
+typedef enum _VMR_CMD_SC_STATUS
```

Comment:
follow the naming convention. Please look at other enums in this file and define them similar